engine/class_modules/sc_shaman.cpp

```diff
@@ -1052,6 +1053,12 @@ struct shaman_action_t : public Base
       ab::energize_type = ENERGIZE_NONE; // disable resource generation from spell data.
     }
 
+    if ( ab::data().affected_by( player -> spec.elemental_shaman -> effectN( 5 ) ) )
+    {
+      ab::base_multiplier *= 1.0 + player -> spec.elemental_shaman -> effectN( 5 ).percent();
+      ab::base_multiplier *= 1.0 + player -> spec.elemental_shaman -> effectN( 7 ).percent();
```

Comment:
This should be the pet aura. I'm not sure if this is the correct way to implement this or not.